locale/ko/Notepad2.rc

```diff
@@ -31,7 +31,7 @@ LANGUAGE LANG_KOREAN, SUBLANG_DEFAULT
 // RT_MANIFEST
 //
 
-IDR_RT_MANIFEST         RT_MANIFEST             "..\\..\\res\\Notepad2.exe.manifest"
+IDR_RT_MANIFEST         RT_MANIFEST             "..\\res\\Notepad2.exe.manifest"

```

Comment:
This change cause build error, see https://ci.appveyor.com/project/zufuliu/notepad2/builds/44828887/job/3u8r75ow4auqr7nv#L223

Reply:
Notepad2.rc(34): error RC2135: file not found: ..\res\Notepad2.exe.manifest [C:\projects\notepad2\locale\ko\Notepad2(ko).vcxproj]